community/kernel/src/main/java/org/neo4j/kernel/impl/newapi/NodeCursor.java

```diff
@@ -58,7 +58,10 @@ void scan( Read read )
         this.next = 0;
         this.highMark = read.nodeHighMark();
         this.read = read;
-        this.labelCursor = read.labelCursor();
+        if ( labelCursor != null )
```

Comment:
I might be missing something here but shouldn't it be `labelCursor == null`

Reply:
Sure looks like it, yeah